src/standard/TIP-4/5.md

```diff
@@ -0,0 +1,113 @@
+
+## Motivation
+The purpose of this standard is to provide an on-chain representation of the `CantBeEvil` license.
+
+The `CantBeEvil` license is made available as a contract that can be inherited by any other contract.
+
+There are six variants of the `CantBeEvil` license:
+
+* [CC0 (“CBE-CC0”)](https://arweave.net/_D9kN1WrNWbCq55BSAGRbTB4bS3v8QAPTYmBThSbX3A/0) – All copyrights are waived under the terms of CC0 1.0 Universal developed by Creative Commons.
+* [Exclusive Commercial Rights with No Creator Retention (“CBE-ECR”)](https://arweave.net/_D9kN1WrNWbCq55BSAGRbTB4bS3v8QAPTYmBThSbX3A/1) – Full exclusive commercial rights granted, with no hate speech termination. Creator does not retain any exploitation rights.
+* [Non-Exclusive Commercial Rights (“CBE-NECR”)](https://arweave.net/_D9kN1WrNWbCq55BSAGRbTB4bS3v8QAPTYmBThSbX3A/2) – Full non-exclusive commercial rights granted, with no hate speech termination. Creator retains exploitation rights.
+* [Non-Exclusive Commercial Rights with Creator Retention & Hate Speech Termination (“CBE-NECR-HS”)](https://arweave.net/_D9kN1WrNWbCq55BSAGRbTB4bS3v8QAPTYmBThSbX3A/3) – Full non-exclusive commercial rights granted, with hate speech termination. Creator retains exploitation rights.
+* [Personal License (“CBE-PR”)](https://arweave.net/_D9kN1WrNWbCq55BSAGRbTB4bS3v8QAPTYmBThSbX3A/4) – Personal rights granted, without hate speech termination.
+* [Personal License with Hate Speech Termination (“CBE-PR-HS”)](https://arweave.net/_D9kN1WrNWbCq55BSAGRbTB4bS3v8QAPTYmBThSbX3A/5) – Personal rights granted, with hate speech termination.
+
+The text of the Licenses is made freely available to the public under the terms of CC0 1.0 Universal. You can also find the full licenses and cover letter in this repo [here](https://github.com/a16z/a16z-contracts/blob/master/licenses).
+
+## Specification
+The keywords “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+### Contracts and interfaces
+* `Collection` - [TIP4.1](1.md) contract that mints tokens;
+* `NFT` - [TIP4.1](1.md) contract that store token information;
+* `CantBeEvil` - contract that is meant to be inherited by NFT contracts and any contract that wishes to expose the `getLicenseURI` and `getLicenseName` methods.
+
+### CantBeEvil
+* Every [TIP-4.1](1.md) `NFT` contract MAY implement instance of `CantBeEvil` contract. `NFT` tokens within the same collection MAY have different license versions unless the version is specifically set in the `Collection` contract;
+* Every [TIP-4.1](1.md) `Collection` contract MAY implement instance of `CantBeEvil` contract. In such case, the derived [TIP-4.1](1.md) `NFT` contract MUST inherit the same license from the collection.
+
+#### `ICantBeEvil.sol`
+```solidity
+pragma ever-solidity >= 0.61.2;
+
+interface ICantBeEvil is TIP_6 {
```

Comment:
I think we need to remove the inheritance of TIP6

Reply:
> I think we need to remove the inheritance of TIP6

Removed inheritance